src/shared/identityUtils.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+
+import fs from 'node:fs';
+import { randomBytes } from 'node:crypto';
+import { DevServerUtils } from './devServerUtils.js';
+
+class LwrConfigFile {
+  public identityToken?: string;
+}
+
+const BYTES_TO_GENERATE = 32;
+
+export class IdentityUtils {
```

Comment:
This is the perfect candidate to be moved to [CryptoUtils.ts or lwc-dev-mobile-core](https://github.com/forcedotcom/lwc-dev-mobile-core/blob/main/src/common/CryptoUtils.ts) later. Please add a `TODO` to this file (similar to what I have [here](https://github.com/salesforcecli/plugin-lightning-dev/blob/main/src/shared/previewUtils.ts#L8-L11)) so that later we refactor it.